gtest/src/error.rs

```diff
@@ -0,0 +1,97 @@
+// This file is part of Gear.
+
+// Copyright (C) 2022 Gear Technologies Inc.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+use anyhow::Error as AnyhowError;
+use codec::Error as CodecError;
+use core_processor::ProcessorError;
+use gear_core::{ids::ProgramId, memory::WasmPageNumber};
+use wasmtime::MemoryAccessError;
+
+pub type TestResult<T> = Result<T, TestError>;
+
+/// List of general errors.
+#[derive(Debug, derive_more::Display)]
+pub enum TestError {
```

Comment:
seems that there is no error case, when we call meta fn on actor, which has no .meta file under the hood

Reply:
You are right, missed this check. Fixed and added a test in https://github.com/gear-tech/gear/pull/1106/commits/10d3f108430f4198ee5b3fc13d4865ccf495d5a5